openfisca_core/formula_toolbox.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+
+from datetime import date  # noqa analysis:ignore
+
+from numpy import maximum as max_, minimum as min_, logical_not as not_, where, select   # noqa analysis:ignore
+
+from .columns import (  # noqa analysis:ignore
+    AgeCol,
+    BoolCol,
+    DateCol,
+    EnumCol,
+    FixedStrCol,
+    FloatCol,
+    IntCol,
+    PeriodSizeIndependentIntCol,
+    StrCol,
+    )
+from .enumerations import Enum  # noqa analysis:ignore
+from .formulas import (  # noqa analysis:ignore
+    ADD,
+    calculate_output_add,
+    calculate_output_divide,
+    dated_function,
+    DIVIDE,
+    set_input_dispatch_by_period,
+    set_input_divide_by_period,
+    missing_value
+    )
+from .base_functions import (   # noqa analysis:ignore
+    requested_period_added_value,
+    requested_period_default_value,
+    requested_period_last_or_next_value,
+    requested_period_last_value,
+    )
+from .variables import DatedVariable, Variable  # noqa analysis:ignore
+from .formula_helpers import apply_thresholds, switch  # noqa analysis:ignore
+from .periods import MONTH, YEAR, ETERNITY  # noqa analysis:ignore
```

Comment:
Otherwise the `flake8` linter doesn't pass as we import stuff we do not use.

And no, there is no way to deactivate a specific error in a specific file. The only options I found are : 
- Deactivating a specific error for the whole project
- Deactivating linting on a given line with `noqa`

@cbenz if you know a better way, I'll happily take it